mediapipe/calculators/ovms/modelapiovmsadapter.hpp

```diff
@@ -50,7 +52,15 @@ class OVMSInferenceAdapter : public ::InferenceAdapter {
     ov::AnyMap modelConfig;
 
 public:
-    OVMSInferenceAdapter(const std::string& servableName, uint32_t servableVersion = 0, OVMS_Server* server = nullptr);
+    OVMSInferenceAdapter(const std::string& servableName, uint32_t servableVersion = 0, OVMS_Server* server = nullptr) :
```

Comment:
Move implementation to cpp so we don't need include header here.

Reply:
We need for windows compilation.